<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <!-- Parent Test Suite -->
    <testsuite name="CalculatorTests" tests="5" failures="1" errors="1" skipped="1" time="0.345">
        
        <!-- First Nested Test Suite -->
        <testsuite name="ArithmeticOperations" tests="3" failures="1" errors="0">
            
            <!-- Successful Test Case -->
            <testcase classname="ArithmeticTests" name="testAddition" time="0.01"/>

            <!-- Failed Test Case -->
            <testcase classname="ArithmeticTests" name="testDivision" time="0.02">
                <failure message="Division by zero" type="java.lang.ArithmeticException">
                    Stack trace details here...
                </failure>
            </testcase>

            <!-- Skipped Test Case -->
            <testcase classname="ArithmeticTests" name="testMultiplication" time="0.00">
                <skipped/>
            </testcase>

        </testsuite>

        <!-- Second Nested Test Suite -->
        <testsuite name="AdvancedOperations" tests="2" failures="0" errors="1">

            <!-- Test Case with Error -->
            <testcase classname="AdvancedTests" name="testSquareRoot" time="0.03">
                <error message="Negative input to sqrt" type="java.lang.MathException">
                    Stack trace details...
                </error>
            </testcase>

            <!-- Successful Test Case -->
            <testcase classname="AdvancedTests" name="testExponentiation" time="0.02"/>
        
        </testsuite>

    </testsuite>
</testsuites>
